content/flux/v0.x/data-types/regexp.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Work with regular expression types
```

Comment:
Notice that basic types and composite types don't have the verbal phrase. Is that intentional?

Reply:
Nope :) nice catch. I'll fix it.